tasks/legacy_release.yaml

```diff
@@ -0,0 +1,70 @@
+---
+- environment:
+    AWS_ACCESS_KEY_ID: "{{ pool_manager_aws_access_key_id }}"
+    AWS_SECRET_ACCESS_KEY: "{{ pool_manager_aws_secret_access_key }}"
+  block:
+    - name: Check if sandbox is already associated
+      vars:
+        _expression: >-
+          available = :a
+          and guid = :g
+          and envtype = :e
+          and attribute_exists(aws_access_key_id)
+          and attribute_exists(aws_secret_access_key)
+          and attribute_exists(hosted_zone_id)
+          and attribute_exists(#Z)
+          and attribute_exists(account_id)
+          and (
+            attribute_not_exists(service_uuid) or service_uuid = :u
+          )
+        _data:
+          ":a":
+            BOOL: false
+          ":g":
+            S: "{{ guid }}"
+          ":u":
+            S: "{{ uuid }}"
+          ":e":
+            S: "{{ env_type }}"
+      command: >-
+        aws
+        --region "{{ pool_region }}"
+        dynamodb scan
+        --table-name {{ pool_table }}
+        --filter-expression '{{ _expression }}'
+        --expression-attribute-values '{{ _data | to_json }}'
```

Comment:
Better to use the `quote` filter than rely on single quotes. `{{ _data | to_json | quote }}`